dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/view/SecretView.java

```diff
@@ -107,7 +111,7 @@ private void buildCommonJson(final AbstractProperty property,
             if (type.equals(Type.BOOL)) {
                 map.put("value", property.getBoolean());
             } else {
-                map.put("value", property.getString());
+                map.put("value", property.isHidden() ? HIDDEN_SECRET_MASK : property.getString());
```

Comment:
I think we can use here a readable value something like: "HIDDEN SECRET", I think it give more information than just "*****"

Reply:
But the requirement says hidden secrets must come back from the resource with that value  "*****" so they can not be read.